sdcm/cluster.py

```diff
@@ -792,6 +793,14 @@ def public_ip_address(self) -> Optional[str]:
             self._public_ip_address_cached = self._get_public_ip_address()
         return self._public_ip_address_cached
 
+    @property
+    def public_dns_name(self) -> str:
+        return self.name
```

Comment:
i'm not sure we'll want to default it like that...

not if we didn't tested it for GCP/Azure/docker

maybe raising not implemented is better for now ?